Packs/McAfee-TIE/Playbooks/playbook-Search_Endpoints_By_Hash_-_TIE.yml

```diff
@@ -102,86 +102,6 @@ tasks:
     quietmode: 0
     isoversize: false
     isautoswitchedtoquietmode: false
-  "2":
-    id: "2"
-    taskid: b098fed0-d1ae-418c-8a37-756e1ae0cb37
-    type: condition
-    task:
-      id: b098fed0-d1ae-418c-8a37-756e1ae0cb37
-      version: -1
-      name: Is ePO enabled?
```

Comment:
Why you removed those tasks?

Reply:
`epo v1` pack has reached its end of life and is no longer supported. End-of-Life Date: Aug 01, 2022
I removed from the playbook the `epo v1` commands.
The condition task for checking if the `McAfee epo v2` integration is enabled and 'epo-find-system (McAfee ePO v2)' command already exists in the playbook.